src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-views/psbt-decoded-request-simple-pairs.tsx

```diff
@@ -0,0 +1,37 @@
+import { InputsOutputPair } from '@app/pages/psbt-request/hooks/match-inputs-and-outputs';
+
+import { PsbtInputOutputPair } from '../psbt-input-output-pair/psbt-input-output-pair';
+import { PsbtInputOutputPlaceholder } from '../psbt-input-output-pair/psbt-input-output-placeholder';
+
+interface PsbtDecodedRequestSimplePairsProps {
+  bitcoinAddressNativeSegwit: string;
+  bitcoinAddressTaproot: string;
+  inputOutputPairs: InputsOutputPair[];
+  showPlaceholder: boolean;
+}
+// ts-unused-exports:disable-next-line
+export function PsbtDecodedRequestSimplePairs({
+  bitcoinAddressNativeSegwit,
+  bitcoinAddressTaproot,
+  inputOutputPairs,
+  showPlaceholder,
+}: PsbtDecodedRequestSimplePairsProps) {
+  if (showPlaceholder) return <PsbtInputOutputPlaceholder />;
+
```

Comment:
Suggest we remove, it's in git history if we need, but hopefully we won't as pairing just doesn't map to how inputs/outputs work.